src/AcceptanceTests/App_Packages/NSB.AcceptanceTests.6.2.0/DelayedDelivery/When_Deferring_a_message.cs

```diff
@@ -29,7 +29,7 @@ public class When_deferring_a_message : NServiceBusAcceptanceTest
                 .Done(c => c.WasCalled)
                 .Run();
 
-            Assert.GreaterOrEqual(context.ReceivedAt - context.SentAt, delay);
+            Assert.That(context.ReceivedAt - context.SentAt, Is.EqualTo(delay).Within(1).Seconds);
```

Comment:
> One test sets the delay to a fraction of the test run times and asserts that the message was received within the test runtime

this of course depends on the delayed delivery mechanism/implementation. But too short delayed delivers are not going through the timeout storage but are short circuited. This might be just a problem with the timeoutmanager though.

> On test sets the delay to a multiple of the test run time and verifies that the message was not received within the test runtime

this would make a very long running test as you would have to account for enough time in the test run so that endpoints are able to spin up before the timeout is reached. Such long running tests is something we wanted to avoid.

@danielmarbach how about raising an issue to discuss this problem in the core?